.github/workflows/cd.yml

```diff
@@ -1,35 +1,28 @@
-# This workflow will build a Java project with Gradle
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
-
 name: CD
 
 on:
   workflow_dispatch:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    env:
-      TEST_AUTH_TOKEN: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
-      TEST_CACHE_NAME: dummy
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout project
+      uses: actions/checkout@v3
       with:
         token: ${{ secrets.MOMENTO_MACHINE_USER_GITHUB_TOKEN }}
-        submodules: true
+
     - name: Set up JDK 16
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v3
       with:
         java-version: 16
```

Comment:
Is there a particular reason we aren't using an LTS version of java for this? I would prefer to switch to 17 at least.